tests/BinaryOperators/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -414,14 +408,13 @@ contract LogicalOperators {
             veryVeryVeryVeryVeryLongVariableCalledB
         ) {}
         if (
-            veryVeryVeryVeryVeryLongVariableCalledA &&
-            veryVeryVeryVeryVeryLongVariableCalledB ||
+            (veryVeryVeryVeryVeryLongVariableCalledA &&
+                veryVeryVeryVeryVeryLongVariableCalledB) ||
```

Comment:
interesting indentation, is that on purpose?

Reply:
yeah, well the indentation rule here falls into the `TupleExpression` indentation rules instead of `BinaryOperation` indentation rules.
Same happens in javascript.
```Javascript
// Before Prettier
if (veryVeryVeryVeryVeryLongVariableCalledA && veryVeryVeryVeryVeryLongVariableCalledB || c);
// After Prettier
if (
  (veryVeryVeryVeryVeryLongVariableCalledA &&
    veryVeryVeryVeryVeryLongVariableCalledB) ||
  c
);
```